### PR TITLE
(Feature) Add new styles to DApp.

### PR DESCRIPTION
**Pull request by gabitoesmiapodo**
**Description:**

Solves https://github.com/poanetwork/poa-dapps-voting/issues/164 by adding the new syles..

**What is it:** (Feature)

**Additional Info:**

I'm attaching a few screen captures to know what to expect from this update.

Core designs:

ballot-tablet-v
new-desktop
new-tablet-v
mobile-menu
new-mobile
new-tablet-h
ballot-desktop
ballot-mobile
ballot-tablet-h
loading

Sokol designs:

new-desktop-sokol
ballot-tablet-v-sokol
new-tablet-v-sokol
mobile-menu-sokol
ballot-desktop-sokol
new-tablet-h-sokol
loading-sokol
ballot-mobile-sokol
ballot-tablet-h-sokol
new-mobile-sokol

### CODE REVIEW

**Comment by varasev** [comment]
The description lines are bound to the upper border of the icon:


image


Maybe add `padding-top: 5px;` for `.ballots .info` to make it look like this?

image

**Comment by gabitoesmiapodo** [comment]
@varasev Fixed it.

**Review by vbaranov — CHANGES_REQUESTED** [review]
@gabitoesmiapodo 
1. Style for ballot creation button should be changed: 'NEW BALLOT' -> 'New Ballot'
2. Key type and Action values in a ballot card should be in lower case with a first capital letter
3. Padding between sliders should be more than padding between a slider and corresponding button

4. 

5. More.../Less buttons should have another color than text. Please see the original layout.

**Comment by gabitoesmiapodo** [comment]
@vbaranov

Good catches, sorry about that. 

- Style for ballot creation button should be changed: 'NEW BALLOT' -> 'New Ballot' -> Fixed.

- Key type and Action values in a ballot card should be in lower case with a first capital letter -> Changed it.

- Padding between sliders should be more than padding between a slider and corresponding button -> Fixed.

- All paddings inside card should be equal. -> Done.

- More.../Less buttons should have another color than text. Please see the original layout. -> Changed that too.

**Comment by vbaranov** [comment]
@gabitoesmiapodo yes, looks good now, thanks. Could you also add small paddings between columns here:



?

**Comment by varasev** [comment]
@gabitoesmiapodo I think the `Load More Ballots` button should be more visible - maybe with some border or background?

**Comment by vbaranov** [comment]
>I think the Load More Ballots button should be more visible - maybe with some border or background?

@varasev there is no borders or another background from designer's layout https://user-images.githubusercontent.com/16716926/43633724-bfffce10-9712-11e8-9a53-527247032d3c.png. I guess, we should follow it.

**Comment by varasev** [comment]
> @varasev there is no borders or another background from designer's layout https://user-images.githubusercontent.com/16716926/43633724-bfffce10-9712-11e8-9a53-527247032d3c.png. I guess, we should follow it.

Maybe make that button just with bold font then? Or make the top and the bottom whitespace around it wider?

**Comment by vbaranov** [comment]
>Maybe make that button just with bold font then? Or make the top and the bottom whitespace around it wider?

maybe, but it should be agreed with an author of the layout https://github.com/poanetwork/poa-dapps-voting/issues/164#issuecomment-410188165.

**Comment by gabitoesmiapodo** [comment]
@vbaranov Added the paddings.

@varasev Maybe we should ask @pashagonchar ?

**Comment by varasev** [comment]
Sorry, didn't guess initially who is designer here :)

**Comment by varasev** [comment]
Ballot description is shown with some words broken near the right border: 

image

Could we turn on a word wrapping?

**Comment by varasev** [comment]
Padding is also needed here:

image

**Comment by varasev** [comment]
`Ballot time` column is in the wrong place for `min threshold` and `change proxy` ballot types. You can find it if you click on `Load More Ballots` several times:

image

image

**Comment by varasev** [comment]
@pashagonchar @gabitoesmiapodo Some little defects with the search bar:

image

image

**Comment by varasev** [comment]
When I reduce the width of the browser window, scroll down a little and click on menu, the menu items are not clickable and they're poorly visible because they're under the faded layout.

Also, I propose not to scroll the header (not to attach it to the top of the browser client area when scrolling) because it occupies a part of the small screen of a smartphone - it brings some discomfort for a user.

image

**Comment by gabitoesmiapodo** [comment]
- Ballot description is shown with some words broken near the right border: Could we turn on a word wrapping? -> Yes, changed it so it's OK now.

- Padding is also needed here: -> Columns should be better separated now.

- Ballot time column is in the wrong place for min threshold and change proxy ballot types. You can find it if you click on Load More Ballots several times: -> OK, went very far into the "Load More" history and I think everything is fine now.

-  Some little defects with the search bar: -> Yeah, those were some states which were being overwritten. I'm not sure if there's something that could be done about the X, but the text is fine now.

- When I reduce the width of the browser window, scroll down a little and click on menu, the menu items are not clickable and they're poorly visible because they're under the faded layout. -> Fixed.

- Also, I propose not to scroll the header (not to attach it to the top of the browser client area when scrolling) because it occupies a part of the small screen of a smartphone - it brings some discomfort for a user. -> We should decide what to do with that. It takes some screen space, sure, but it also makes the menu available at all times. It could be very far away if the user loads more ballots a few times, and he / she should have to scroll up **a lot** if they reach the bottom.

**Comment by varasev** [comment]
@gabitoesmiapodo Thanks for the fixes!

> We should decide what to do with that. It takes some screen space, sure, but it also makes the menu available at all times...

Ok, let's leave it as is.

Could you also make another ballot card that was added in the layout here: https://github.com/poanetwork/poa-dapps-voting/issues/164#issuecomment-412256495

We didn't yet implement this type of ballots in DApp, but you could add makeup of it in the `render` function of `src/components/BallotCard.jsx` just for a demo to see if it looks correct. Then we will comment out its code until the logic is implemented.

**Comment by varasev** [comment]
@gabitoesmiapodo I mean this:

poa_governance_app_sokol_2

**Comment by gabitoesmiapodo** [comment]
@varasev 

Now the "Load more" button looks as you asked in https://github.com/poanetwork/poa-dapps-voting/issues/164#issuecomment-412537680

Working on https://github.com/poanetwork/poa-dapps-voting/pull/166#issuecomment-412428620 now.

**Comment by gabitoesmiapodo** [comment]
@varasev 

What was asked in https://github.com/poanetwork/poa-dapps-voting/pull/166#issuecomment-412428620 should be done by now.

Testing code is in the components/BallotCard.jsx file, lines 445 to 503.

Original (No / Yes) code is commented out from line 505 to 541.

When this is tested / approved we should uncomment 505 to 541, and comment out 445 to 503.

**Comment by varasev** [comment]
@gabitoesmiapodo ok, thank you. We also need to have the next columns for the new type of ballot:
- Proposer
- Proposed Receiver
- Funds Amount
- Ballot Time

as it's shown in https://github.com/poanetwork/poa-dapps-voting/pull/166#issuecomment-412428620

**Review by vbaranov — APPROVED** [review]

**Comment by gabitoesmiapodo** [comment]
@varasev OK, there it is now.

Original code is commented out in src/components/BallotKeysCard.jsx lines 39 to 65.

new columns

File: src/assets/stylesheets/application/ballots/_about.scss [hunk 1]
```diff
@@ -22,6 +22,9 @@
       padding-right: $cell-right-padding;
     }
     &_name {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
```
Comment (varasev) [inline src/assets/stylesheets/application/ballots/_about.scss]: @gabitoesmiapodo Let's return the word wrapping for the `name` (`proposer`) column. The previous look was better.

**Review by varasev — CHANGES_REQUESTED** [review]

**Comment by gabitoesmiapodo** [comment]
@varasev OK, done.

**Comment by varasev** [comment]
@gabitoesmiapodo Ok, let's comment out the code for new ballot type and uncomment the original code. And then we'll merge all the changes.

**Comment by gabitoesmiapodo** [comment]
@varasev Done.

**Comment by pashagonchar** [comment]
@gabitoesmiapodo change logo please
logo_1
logo_2

https://zpl.io/VxlmwQ3

**Comment by gabitoesmiapodo** [comment]
@pashagonchar Changed.

**Review by varasev — APPROVED** [review]

**Pull request merged**